galley/docker/Dockerfile.galley

```diff
@@ -10,4 +10,15 @@ FROM gcr.io/distroless/static as distroless
 # This will build the final image based on either default or distroless from above
 FROM ${BASE_DISTRIBUTION}
 ADD galley /usr/local/bin/
+
+# 1001 is chosen as the UID to be added to the istio group as this is the UID that galley in the docker container runs as.
+RUN addgroup --system --gid 1001 istio && \
+    adduser --system --uid 1001 --gid 1001 istio && \
+    chown -R 760 /usr/local/bin/galley
+
+# allow binding a socket to Internet domain privileged ports
+RUN setcap cap_net_bind_service=+eip /usr/local/bin/galley
```

Comment:
setcap on a file is not portable between platforms.  Pleae find a different way. :)

Cheers
-steve

Reply:
I have investigated another way to add `sysctls` in `securityContext` for pod:
```
securityContext:
        sysctls:
        - name: net.ipv4.ip_unprivileged_port_start
          value: "0"
```
`net.ipv4.ip_unprivileged_port_start` is an unsafe sysctls, it requires  to enable it on a node-by-node. refer to this official kubernetes document - https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#enabling-unsafe-sysctls
I think that should not an acceptable solution for us.